scripts/gen_profiles_solution.py

```diff
@@ -1421,7 +1455,7 @@ def __init__(self, xml, upperCaseName):
         self.limits = dict()
         self.platform = xml.get('platform')
         self.provisional = xml.get('provisional')
-        self.promotedTo = xml.get('promotedto')
+        self.promotedTo = xml.get('promotedto').split(',') if xml.get('promotedto') is not None else []
```

Comment:
Is there a case where an extension is promoted to multiple extensions? Or is this for promotedto an extension and a core version maybe?